app/Resources/TwigBundle/views/Exception/error503.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends '::_base.html.twig' %}
+
+
+{% block body %}
+    <h1>Oops! An Error Occurred</h1>
+    <h2>The server returned a "{{ status_code }} {{ status_text }}".</h2>
+
+    <div>
+        {{ exception.message }}
```

Comment:
The content of the message should be safe to show to users (it comes from a `ServiceUnavailableHttpException`). The whole exception is not.